packages/core/src/routes/connector.ts

```diff
@@ -75,8 +74,7 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         await connectorInstance.validateConfig(body.config);
       }
 
-      await updateConnector({ set: body, where: { id } });
-      ctx.body = transpileConnectorInstance(await getConnectorInstanceById(id));
+      ctx.body = await updateConnector({ set: body, where: { id } });
```

Comment:
```
const connector = await updateConnector({ set: body, where: { id } });
ctx.body = transpileConnectorInstance(connector);
```

Reply:
then we should align the usage of "connector" in this router. either all of them are just db entry operations, either are full instance operations.